paasta_tools/setup_kubernetes_cr.py

```diff
@@ -243,6 +263,9 @@ def format_custom_resource(
         },
         "spec": instance_config,
     }
+    dashboard_url = get_dashboard_url(kind, service, instance, cluster)
+    if dashboard_url:
+        resource["metadata"]["labels"]["yelp.com/dashboard_url"] = dashboard_url
```

Comment:
yeah, I'm not sure we should change the labels here: we run the risk of masking the 'real' labels, and the users won't be sure of which are from kubernetes and which are from paasta.

instead, I'd just append to the metadata dictionary

Reply:
you cannot add arbitrary info into metadata itself, it's just a structure defined inside k8s: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L110
Instead os that please use `Annotations` map which is there for arbitrary data: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L238

Reply:
Agree, I will store this in annotations. Thank you!